www-apps/radicale/radicale-2.1.12.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Please test, if python3.8 and python3.9 is supported.

Reply:
Tested against python3.8, works. No test and no time to test against python3.9 yet.
So I'd keep python3.7 + python3.8 for now.

Reply:
One more on python3.9 compatibility:

One important way to run the radicale server is plugging it to apache via
mod_wsgi. However, mod_wsgi has no python3.9 support yet, only python3.8.

mod_wsgi has to be recompiled against the same python single target as
radicale. mod_wsgi with python3.7 complains when run against radicale using
python3.8.

So yeah, I left radicale on python3.7 and python3.8.

Reply:
If a user wants to use mod_wsgi with radicale they can ensure their PYTHON_SINGLE_TARGETs are the same. mod_wsgi isn't a dependency of radicale, so radicale's PYTHON_COMPAT can be bumped without actually hurting mod_wsgi.

Reply:
> If a user wants to use mod_wsgi with radicale they can ensure their
> PYTHON_SINGLE_TARGETs are the same. mod_wsgi isn't a dependency of radicale,
> so radicale's PYTHON_COMPAT can be bumped without actually hurting mod_wsgi.

Last time I checked radicale as a stand alone server (without apache/mod_wsgi)
I stubled accross some significant security issues, mentioned by the
developer himself.

So, I am close to saying mod_wsgi *is* kind of a dependency (while of couse
not beeing an ideal solution, either).

At least, I have no testbed without mod_wsgi and personally would prefer to
wait for mod_wsgi to become compatible with python3.9 so I can test radicale
against python3.9, too.

Let me know in case there are reasons to hurry up, though.

Reply:
If mod_wsgi is a dependency, why it's not listed in the ebuild? You can also add it as `optfeature`.